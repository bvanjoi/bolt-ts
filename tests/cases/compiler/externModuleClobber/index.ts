// From `github.com/microsoft/TypeScript/blob/v5.7.3/tests/cases/compiler/externModuleClobber.ts`, Apache-2.0 License

declare module EM {
	export class Position { }

	export class EC {
		public getPosition() : EM.Position;
	}
}

var x:EM.Position;
var ec:EM.EC = new EM.EC();

x = ec.getPosition();

var y: string = ec.getPosition();
//~^ ERROR: Type 'Position' is not assignable to type 'string'.