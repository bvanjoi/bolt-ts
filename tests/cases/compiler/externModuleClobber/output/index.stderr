  × Type 'Position' is not assignable to type 'string'.
    ╭─[../../tests/cases/compiler/externModuleClobber/index.ts:16:5]
 16 │ y
    · ─
    ╰────
